addons/base_vat/models/res_partner.py

```diff
@@ -215,6 +215,14 @@ def _run_vat_test(self, vat_number, default_country, partner_is_company=True):
         # If vat is set before country_id, the constraint must not break.
         return check_result
 
+    def _get_vat_str(self):
+        country = self.commercial_partner_id.country_id
+        if country.code == 'IN':
+            vat_str = country.vat_label or _('Tax ID')
```

Comment:
yes, because it's not breaking standard policy.
and good to have it

Reply:
You are doing something very country specific for India in base_vat...  Maybe applying VAT label all the time is not bad: your RUC number, your Tax ID number, your GSTIN number, your PIN number, ...  And that can fallback on _('Tax ID') Why not?  (but changes of behaviour should be avoided in stable if it is not blocking the user, so you are still on the edge...)

Reply:
I was thinking Belgium is problematic, but it is actually not, because the VAT label field is translate True: VAT, BTW, TVA will work.   (so you can do this for all countries)